dom.bs

```diff
@@ -5734,6 +5735,12 @@ null if <var>event</var>'s <a>composed flag</a> is unset and <a for=/>shadow roo
 <p>The <dfn attribute for=ShadowRoot><code>mode</code></dfn> getter steps are to return
 <a>this</a>'s <a for=ShadowRoot>mode</a>.</p>
 
+<p>The <dfn attribute for=ShadowRoot><code>delegatesFocus</code></dfn> getter steps are to return
+<a>this</a>'s <a for=ShadowRoot>delegates focus</a>.</p>
+
+<p>The <dfn attribute for=ShadowRoot><code>slotAssignment</code></dfn> getter steps are to return
```

Comment:
I changed the order to match the IDL here.

Reply:
Thanks! (Here I thought I could keep blame clean by using the modern wording when landing slot assignment. There's always further nits. 😊)